hrl_dorm_apt.md

```diff
@@ -127,7 +127,7 @@ MB由Tower和East Building (也有人习惯称为East Wing) 组成。单人间
 
 * 饭堂:  
 MB的饭堂算是校内饭堂前三名了（然而吃了一两个月还是会吃腻），位于 East Building 负一楼。  
-Dining 网页：[http://www.dining.umn.edu/CampusRestaurants/ResidentialRestaurants/middlebrook-wb.html](http://www.dining.umn.edu/CampusRestaurants/ResidentialRestaurants/middlebrook-wb.html)
+Dining 网页：[http://www.dining.umn.edu/CampusRestaurants/ResidentialRestaurants/middlebrook-wb.html](https://dining.umn.edu/dininghalls/middlebrook)
```

Comment:
The link description can be changed to [https://dining.umn.edu/dininghalls/middlebrook] as well